index.html

```diff
@@ -9,6 +9,32 @@
   <body class="page">
     <div class="page__container">
       <p>The repository is created with the purpose of learning adaptive web design and team collaboration using Git and GitHub.</p>
+      <section class="contacts">
+        <h2 class="contacts__title">Остались вопросы?Свяжитесь с нами!</h2>
+        <ul class="contacts__list">
+          <li class="contacts__item">
+            <h3 class="contacts__heading" >Звоните:</h3>
+            <p class="contacts__access">+7(812) 555-66-66</p>
```

Comment:
Similarly, we can make our phone number a link with <a> tag with the `tel` attribute.

```html
<a href="tel:+12345678900">+1 (234) 567-89-00</a>
```
This change enables users to easily call the provided phone number by clicking on it, providing a more convenient and intuitive interaction.

Reply:
@naumch1k  I did it. But then "a" started to underline. So I added _text-decoration:none_ in the CSS and now it looks good.

Reply:
@SergeyAlexandrowich Link tag indeed comes with some default styles including underline. Great job noticing and overwriting styles you don't need!